images/linux/scripts/tests/Common.Tests.ps1

```diff
@@ -37,6 +37,10 @@ Describe "Swift" -Skip:(Test-IsUbuntu22) {
     It "swiftc" {
         "swiftc --version" | Should -ReturnZeroExitCode
     }
+
+    It "libsourcekitd" {
+        "test -f /usr/local/lib/libsourcekitdInProc.so" | Should -ReturnZeroExitCode
```

Comment:
nitpick:  we should use `-Exist` param to check the lib path without using test.

Reply:
:+1: updated